tools/image-info

```diff
@@ -499,9 +499,6 @@ def append_ostree_repo(report, repo):
     resolved = {r: ostree("rev-parse", r).stdout.strip() for r in refs}
     commit = resolved[refs[0]]
 
-    refs = {r: {"inputhash": ostree("show", "--print-metadata-key=rpmostree.inputhash", resolved[r]).stdout.strip("'\n")} for r in refs}
-    report["ostree"]["refs"] = refs
-
```

Comment:
Sad to see this go, be neat to (in the future) have a `--complete` flag or so for `image-info` to get info that are not meant for tests, or something like this.

Reply:
Yeah, this would be nice I think. :)